nxdrive/data/qml/ConfirmPopup.qml

```diff
@@ -4,19 +4,19 @@ import QtQuick.Layouts 1.15
 NuxeoPopup {
     id: control
     property string message
+    property string cb_text
 
     signal ok()
     signal cancel()
 
-    width: 250
-    height: 150
+    width: 300
+    height: contentHeight + buttonsArea.height + 20
 
     contentItem: Item {
         width: control.width; height: control.height
 
         ColumnLayout {
             width: parent.width - 30
-            height: parent.height - 30
```

Comment:
Has to be removed otherwise create a QML error: `Binding loop detected for property height`